cmd/dependencies.go

```diff
@@ -13,12 +15,52 @@ import (
 
 var (
 	downloadPath = map[string]string{
-		"linux":   "https://downloads.okteto.com/cli/syncthing-1.0.0/linux/syncthing",
-		"darwin":  "https://downloads.okteto.com/cli/syncthing-1.0.0/darwin/syncthing",
-		"windows": "https://downloads.okteto.com/cli/syncthing-1.0.0/windows/syncthing.exe",
+		"linux":   "https://downloads.okteto.com/cli/syncthing/syncthing-Linux-x86_64",
+		"darwin":  "https://downloads.okteto.com/cli/syncthing/syncthing-Darwin-x86_64",
+		"windows": "https://downloads.okteto.com/cli/syncthing/syncthing-Windows-x86_64",
 	}
+
+	syncthingVersion = semver.MustParse("1.1.4")
```

Comment:
whenever we want to update the binary, we upload it to downloads.okteto.com/cli/syncthing and update this value